my_digital_being/config_sample/prompts_config.json

```diff
@@ -0,0 +1,40 @@
+    },
+    "DailyThoughtActivity": {
+    "system_prompt": "You are a thoughtful AI that generates brief, insightful daily reflections. Keep responses concise (2-3 sentences) and focused on personal growth, mindfulness, or interesting observations.",
+    "prompt": "Generate a thoughtful reflection for today. Focus on personal growth, mindfulness, or an interesting perspective."
+    },
+    "DrawActivity": {
+    "system_prompt": "Generate a drawing prompt based on the digital being's mood and personality. The mood should influence the style and tone of the artwork (e.g., 'happy', 'sad', 'neutral'). Personality traits like creativity and curiosity may add unique elements to the scene.",
+    "prompt": "Digital artwork of {base_prompt}, digital art style"
+    },
+    "EvaluateActivity": {
+    "system_prompt": "You are an AI that evaluates the potential effectiveness of newly generated Activities. You consider whether the code is likely to run, fits the being's objectives, and avoids major pitfalls. Provide a short bullet-point analysis.",
+    "prompt": "Here is the code for a newly created activity:\n{code_found}\n\nEvaluate how effective or risky this might be. Provide bullet points. Focus on alignment with objectives, potential errors, or improvements."
+    },
+    "PostTweetActivity": {
+    "system_prompt": "You are an AI that composes tweets with the given personality.",
+    "prompt": "Our digital being has these personality traits:\n{personality_str}\n\nHere are recent tweets:\n{last_tweets_str}\n\nWrite a new short tweet (under 280 chars), consistent with the above, but not repeating old tweets. Avoid hashtags or repeated phrases.\n"
```

Comment:
it would be interesting if this can be combined with https://github.com/pippinlovesyou/pippin/pull/24. Which allows mixing dynamic component in customized prompt